tests/insert-pragma/graphql/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,27 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`with-pragma.graphql 1`] = `
+# @format
+
+type Foo {
+  one: Type
+}
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# @format
+
+type Foo {
+  one: Type
+}
+
+`;
+
+exports[`without-pragma.graphql 1`] = `
+type Foo {
+  one: Type
+}
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+type Foo {
```

Comment:
Looks like the pragma isn’t getting inserted here.

Reply:
Yeah I didn't write `insertPragma` for GraphQL yet, I asked for help with this in OP